components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -195,12 +195,12 @@ internal sealed class SubscriptionWrapper : ISubscription {
                     if (_subscription?.Configuration?.ResolveDisplayName ?? false) {
                         ResolveDisplayNameAsync(_subscription.MonitoredItems);
                     }
-                    await ReapplyAsync(session, false);
+                    await ReapplyAsync(session, false).ConfigureAwait(false);
                     Enabled = true;
                     Active = false;
                 }
                 catch (Exception e) {
-                    _logger.Error(e, "Failed to enable subscription");
+                    _logger.Verbose(e, "Failed to enable subscription");
```

Comment:
>Verbose [](start = 28, length = 7)

generally for exceptions I think is a good idea to make such exceptions visible. In this particular case we do not expect exceptions and if they are caught something went seriously bad and we want to know ... the logic here was totally changed and we are no longer relying on exceptions

Reply:
I will revert this, only 10 hits in 10 min

Reply:
ok, that's wrong we need to investigate why ...